app/models/equipment_model.rb

```diff
@@ -222,13 +222,11 @@ def model_restricted?(reserver_id)
   end
 
   def available_item_select_options
-    # TODO: this is apparently a security risk
-    equipment_items.includes(:reservations).active.select(&:available?)\
-                   .sort_by(&:name)\
-                   .collect do |item|
-                     "<option value=#{item.id}>#{item.name}</option>"
-                   end\
-                   .join.html_safe
+    @items ||= equipment_items.includes(:reservations).active
+                              .select(&:available?).sort_by(&:name)
+    @str ||= sanitize(@items.collect do |item|
+                        "<option value=#{item.id}>#{item.name}</option>"
+                      end.join)
```

Comment:
this should go in a view helper, not the model (travis is failing because `sanitize` isn't a method on EquipmentModel, which makes sense)